src/lib/utils/envHelper.ts

```diff
@@ -168,12 +174,24 @@ export const getCurrentNetwork = () => {
     const networkId = NETWORKS_DATA[value].networkId;
     return { ...NETWORKS_DATA[value], ...NETWORK_VARIABLES[networkId] };
   }
-  window.localStorage.setItem('current_network', '0');
-  const networkId = NETWORKS_DATA[0].networkId;
-  return { ...NETWORKS_DATA[0], ...NETWORK_VARIABLES[networkId] };
+  if (!value && process.env.REACT_APP_FAUCET_PAGE) {
+    const holeskyIndex = NETWORKS_DATA.findIndex((networkData: any) => networkData.networkId === NETWORKS.HOLESKY);
+    return saveNetwork(holeskyIndex);
+  }
+  return saveNetwork(0);
```

Comment:
What is the meaning of 0? Introduce a const for it to make it clearer

Reply:
is meaning index in array of networks, by default, when u open ssv for your first time you don't have this option in local storage so it's just open the first network

Reply:
@alexstolr it's not some specify network, it's only the index, we just take a first network in networks array, in prod it's mainnet, on stage we have only one for now, it's holesky, anyway I moved this 0 as constant and deliver this change in next merge